Cargo.toml

```diff
@@ -1,20 +1,8 @@
 [workspace]
-members = [
-    "bin/ream",
-    "crates/common/consensus",
-    "crates/common/execution_engine",
-    "crates/common/executor", 
-    "crates/common/network_spec",
-    "crates/crypto/bls",
-    "crates/networking/discv5",
-    "crates/networking/p2p",
-    "crates/rpc",
-    "crates/runtime",
-    "crates/storage",
-    "testing/ef-tests",
-]
 
 default-members = ["bin/ream"]
+members = ["bin/ream", "crates/common/consensus", "crates/common/execution_engine", "crates/common/executor", "crates/common/network_spec", "crates/crypto/bls", "crates/networking/discv5", "crates/networking/p2p", "crates/rpc", "crates/runtime", "crates/storage", "testing/ef-tests",
+]
```

Comment:
Is there a way to make sort flatten these? as ideally they aren't scrunched up like this